aiodocker/images.py

```diff
@@ -61,11 +63,27 @@ def __init__(self, docker):
         if tag:
             params['tag'] = tag
 
+        headers = {"content-type": "application/json"}
+
+        if auth and 'auth' in auth:
```

Comment:
What do you think about moving the code to utils as a function?

I think it will be cleaner.